src/Data/Text/IO/Utf8.hs

```diff
@@ -0,0 +1,104 @@
+-- |
+-- Module      : Data.Text.IO.Utf8
+-- Copyright   : (c) 2009, 2010 Bryan O'Sullivan,
+--               (c) 2009 Simon Marlow
```

Comment:
It feels weird to copy a decade-old copyright into new files. It seems the content is all new?.. Unless I'm missing something, you can probably omit the field entirely or put your own name if you wish.